AudioBar/AudioBarTests.swift

```diff
@@ -355,6 +374,69 @@ class AudioBarTests: XCTestCase, Tests {
         expect(view?.playbackTime, "-0:40")
     }
 
+    func testPlaybackDuration1() {
+        let view = expectView(for: .readyToPlay(.init(duration: 1)))
+        expect(view?.playbackDuration, 1)
+    }
+
+    func testPlaybackDuration2() {
+        let view = expectView(for: .readyToPlay(.init(duration: 20)))
+        expect(view?.playbackDuration, 20)
+    }
+
+    func testPlaybackDuration3() {
+        let view = expectView(for: .readyToPlay(.init(duration: 21)))
+        expect(view?.playbackDuration, 21)
+    }
+
+    func testPlaybackDuration4() {
+        let view = expectView(for: .readyToPlay(.init(duration: 60)))
+        expect(view?.playbackDuration, 60)
+    }
+
+    func testElapsedPlaybackTime1() {
```

Comment:
Why do we have 4 tests for `nil`, 0, 1 and 20 seconds? We need just three — `nil`, 1 and 2 seconds.